test/threadsafe_function/threadsafe_function.cc

```diff
@@ -0,0 +1,85 @@
+#define NAPI_EXPERIMENTAL
```

Comment:
Does `NAPI_EXPERIMENTAL` still required? Maybe a more specific one `#define NAPI_VERSION 4` is preferred?